webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/smartling/quartz/SmartlingPullBatchFileJobInput.java

```diff
@@ -0,0 +1,94 @@
+package com.box.l10n.mojito.service.thirdparty.smartling.quartz;
+
+import com.box.l10n.mojito.entity.Repository;
+import com.box.l10n.mojito.service.thirdparty.ThirdPartyTMSSmartling;
+import com.box.l10n.mojito.service.thirdparty.smartling.SmartlingOptions;
+import com.box.l10n.mojito.smartling.SmartlingClient;
+import java.util.Map;
+
+public class SmartlingPullBatchFileJobInput {
+
+  Long batchNumber;
+  String pluralSeparator;
+  Repository repository;
```

Comment:
it sounds like we're passing an entity as job input. It may create all sorts of issue with serialization/deserialization. I think it is better to stick to simple types.